File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{ int n;

    do
    {
       n = get_int("Height?:\n");
    }
    while (n < 1 || n > 8); 
    
    
    for (int x = 0; x < n; x++)
    { 
        for (int z = 1; z < (n-x); z++)
        {printf(" ");
        }
        for (int y = 0; y <= x; y++)
        {printf("#");
        }
    
        printf("\n");
    }
        
    }

File: mario.py
from cs50 import get_int


height = get_int("Height: ")
while height > 8 or height < 1:
    height = get_int("Height: ")
for j in range(height):
    print(" " *(height - (j + 1)), end="" )
    print("#" * (j + 1))